### PR TITLE
[docs] fix make-docs

### PR DESCRIPTION
**Pull request by akotlar**
Addresses https://github.com/hail-is/hail/issues/8940

### CODE REVIEW

File: hail/python/hail/docs/conf.py [hunk 1]
```diff
@@ -224,7 +224,14 @@
 #
 www_root = os.path.dirname(os.path.abspath(__file__))
 _i = 0
-while _i < 50 and not os.path.isdir(os.path.join(www_root, 'www')):
+while _i < 50:
+    if(os.path.isdir(os.path.join(www_root, 'www'))):
+        break
+
+    if(os.path.isdir(os.path.join(www_root, 'site'))):
```
Comment (akotlar) [inline hail/python/hail/docs/conf.py:231]: since build depends on www we need 2 paths, or a much larger commit in which we rename site to www

Comment (danking) [reply hail/python/hail/docs/conf.py:231]: These files aren't actually referenced in the docs. The docs refer directly to `/navbar.css`. It seems like we should just remove lines 225 through 228 and reduce `html_extra_path` to just `misc/`. If we want the navbar to be present when viewing the docs locally, we should create a `hail/Makefile` target that sets up the environment correctly or a `site/Makefile` target that runs the whole site locally.

IMO, when I look at the docs locally it's just to check that I got my RST syntax right, I don't think the nabber is so important to that.

Comment (danking) [reply hail/python/hail/docs/conf.py:231]: I did try to get a local site working correctly but it was kind of annoying to parameterize nginx's TLS setting and I found `make -C site deploy NAMESPACE=dking` pretty fast anyway

Comment (danking) [reply hail/python/hail/docs/conf.py:231]: But I'm definitely here for a good local site preview.

**Comment by akotlar** [comment]
> These files aren't actually referenced in the docs. The docs refer directly to /navbar.css

This is used in layout.html. Doc shares the site navbar.

File: hail/python/hail/docs/conf.py (hunk 1 above)
Comment (akotlar) [reply hail/python/hail/docs/conf.py:231]: Can we use python -m http.server? I don't think nginx is needed for site.

**Comment by danking** [comment]
I'm not sure I understand. `hail/python/hail/docs/_templates/layout.html` references `/navbar.css` which should be present on the deployed site. I might misunderstand `conf.py`, but, AFAICT, this makes unused copies of navbar.css and the PNG: https://hail.is/docs/0.2/navbar.css and https://hail.is/docs/0.2/hail-logo-cropped.png. The page at https://hail.is/docs/0.2/ loads `hail.is/navbar.css`.

**Comment by akotlar** [comment]
> I'm not sure I understand. `hail/python/hail/docs/_templates/layout.html` references `/navbar.css` which should be present on the deployed site. I might misunderstand `conf.py`, but, AFAICT, this makes unused copies of navbar.css and the PNG: https://hail.is/docs/0.2/navbar.css and https://hail.is/docs/0.2/hail-logo-cropped.png. The page at https://hail.is/docs/0.2/ loads `hail.is/navbar.css`.

Sorry, ignore, I misread the comment. Agreed

**Comment by akotlar** [comment]
There are also bootstrap files used in the project, but no longer in site, so those can be moved over.

**Pull request merged**